engine/apps/slack/scenarios/paging.py

```diff
@@ -356,20 +357,25 @@ def render_dialog(slack_user_identity, slack_team_identity, payload, initial=Fal
 
     blocks.extend([_get_title_input(payload), _get_message_input(payload)])
 
-    view = _get_form_view(submit_routing_uid, blocks, json.dumps(new_private_metadata))
+    view = _get_form_view(submit_routing_uid, blocks, json.dumps(new_private_metadata), selected_organization)
     return view
 
 
-def _get_form_view(routing_uid, blocks, private_metatada):
+def _get_form_view(routing_uid, blocks, private_metatada, organization):
+    try:
+        channel = organization.slack_team_identity.get_cached_channels().get(
+            slack_id=organization.general_log_channel_id
+        )
+        additional_info = f":information_source: The alert group will be posted to the #{channel.name} Slack channel"
+    except SlackChannel.DoesNotExist:
+        additional_info = (
+            ":information_source: The alert group will be posted to the default Slack channel if there is one setup"
```

Comment:
wouldn't a `SlackChannel.DoesNotExist` exception mean that there is no default slack channel configured for the organization? If that is the case, should the wording on this message be changed (or maybe even excluded)?

Reply:
There could be a channel setup for which we don't have the cached information yet (unlikely, but still, when the exception is raised, I think the displayed message is useful to know that you can setup a default channel and it will be used in that case)

Reply:
makes sense 👍